awx/conf/apps.py

```diff
@@ -11,8 +11,16 @@ class ConfConfig(AppConfig):
     name = 'awx.conf'
     verbose_name = _('Configuration')
 
+    def configure_oauth2_provider(self, settings):
+        from oauth2_provider import settings as o_settings
+        o_settings.oauth2_settings = o_settings.OAuth2ProviderSettings(
+            settings.OAUTH2_PROVIDER, o_settings.DEFAULTS,
+            o_settings.IMPORT_STRINGS, o_settings.MANDATORY
+        )
+
     def ready(self):
         self.module.autodiscover()
         from .settings import SettingsWrapper
         SettingsWrapper.initialize()
         configure_external_logger(settings)
+        self.configure_oauth2_provider(settings)
```

Comment:
Maybe do this under the `sso` app, since it's for that purpose and not configuration in general?

Reply:
The load sequence of apps `sso` and `conf` might be the reason for this placement.